content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/start.rb

```diff
@@ -30,6 +31,17 @@ def service
                 end
               end
 
+              def check_interval
+                return unless @handle.root['ae_retry_ttl']
```

Comment:
@tinaafitz @billfitzgerald0120 This logic is not taking the current generic state-machine `action` into account.  Is this value being populated during an automate instance resolution?

If the value is coming straight from the service_template (or service) config settings I would not expect it to have the `ae_` prefix.